src/redux/books/books.js

```diff
@@ -39,9 +29,64 @@ export const addBook = (payload) => ({
   payload,
 });
 
+export const sentBook = (payload) => async (dispatch) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: payload.id,
+      title: payload.title,
+      author: payload.author,
+
+      // title: {
+      //   bookTitle: payload.title,
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you remove commented-out code from your codebase as it doesn't make your code neat.